app/presenters/schools/users/show_presenter.rb

```diff
@@ -0,0 +1,112 @@
+module Schools
+  module Users
+    class ShowPresenter < ApplicationPresenter
+      attr_reader :user
+
+      def initialize(view_context, user)
+        @user = user
+        super(view_context)
+      end
+
+      def avatar_url
+        user.avatar_url || user.initials_avatar
```

Comment:
Don't use `User#initials_avatar`. Return `nil`, and conditionally use the `Avatar` client-side component instead in the view; it has support for server-rendered pages.

We really should mark the `initials_avatar` method as deprecated, similar to how `image_or_avatar_url` is deprecated.